### PR TITLE
Feature/nwps 220 reallow card groups on landing

### PR DESCRIPTION
**Pull request by Raq-HEE**
Hi @smanivasag - can you give this a review please. This adds cards to bottom of guidance pages and minihub, removes the groovy script, and adds cards to blog pages.
Thanks,
Raq

### CODE REVIEW

File: repository-data/webfiles/src/main/resources/site/freemarker/hee/macros/guidance-content.ftl [hunk 1]
```diff
@@ -96,6 +96,9 @@
                             </#list>
                         </div>
                     </#if>
+                    <div class="nhsuk-grid-column-full nhsuk-section__content">
```
Comment (smanivasag) [inline repository-data/webfiles/src/main/resources/site/freemarker/hee/macros/guidance-content.ftl:99]: Could we wrap this up with `<#if guidanceDocument.relatedContent??>...</#if>` as `guidanceDocument.relatedContent` is an optional field ?

File: repository-data/webfiles/src/main/resources/site/freemarker/hee/pages/minihub-main.ftl [hunk 2]
```diff
@@ -81,6 +81,9 @@
                         </#if>
                     </ul>
                 </nav>
+                <div class="nhsuk-width-container">
```
Comment (smanivasag) [inline repository-data/webfiles/src/main/resources/site/freemarker/hee/pages/minihub-main.ftl:84]: Again, could we wrap this up with `<#if document.relatedContent??>...</#if>` as `guidanceDocument` is an optional field ?

**Review by smanivasag — COMMENTED** [review]
HI @Raq-HEE - Thanks for the PR and it looks good. Could you take a look at the couple of review comments ? thanks

File: repository-data/webfiles/src/main/resources/site/freemarker/hee/macros/guidance-content.ftl (hunk 1 above)
Comment (Raq-HEE) [reply repository-data/webfiles/src/main/resources/site/freemarker/hee/macros/guidance-content.ftl]: done.

File: repository-data/webfiles/src/main/resources/site/freemarker/hee/pages/minihub-main.ftl (hunk 2 above)
Comment (Raq-HEE) [reply repository-data/webfiles/src/main/resources/site/freemarker/hee/pages/minihub-main.ftl]: done.

**Review by smanivasag — APPROVED** [review]

**Pull request merged**